test/Scripts.Integration.Test/Editor/Builder.cs

```diff
@@ -17,11 +17,23 @@ public static void BuildIl2CPPPlayer(BuildTarget target, BuildTargetGroup group)
         EditorUserBuildSettings.selectedBuildTargetGroup = group;
         PlayerSettings.SetScriptingBackend(group, ScriptingImplementation.IL2CPP);
 
-        if (args.ContainsKey("sentryOptions.configure"))
+        var cliOptions = AssetDatabase.LoadAssetAtPath<SentryCliOptions>(Path.Combine("Assets", "Plugins", "Sentry", "SentryCliOptions.asset"));
+
+        // 'build-project.ps1' explicitely calls for uploading symbols
+        if(args.ContainsKey("uploadSymbols"))
+        {
+            Debug.Log("Enabling automated debug symbol upload.");
+            cliOptions.UploadSymbols = true;
+        }
+        else
         {
-            SetupSentryOptions(args);
+            Debug.Log("Disabling automated debug symbol upload.");
+            cliOptions.UploadSymbols = false;
```

Comment:
What does this solve? UploadSymbols defaults to `true`

Reply:
Since it is `true` by default there is no way of running it locally. The build would always fail with sentry-cli trying to upload to a non-running server.

Reply:
Can you maybe write down the scenario step-by-step when this doesn't work, so maybe we can figure out a fix without more exceptional cases in scripts?

Reply:
So the way I had it running:
- `pwsh create-project.ps1 'unityPath'`
- `pwsh update-sentry.ps1 'unityPath`
- `pwsh build-project.ps1 'unityPath'`

And `build-project` would fail the build. I was running it without `-CheckSymbols` but the upload is enabled by default. `sentry-cli` would fail to upload due to dummy values.

Reply:
Why not just call `pwsh build-project.ps1 'unityPath' -CheckSymbols` instead? That's how it's supposed to be run :)

But yeah, I get your point, let me check the other places you had to change to make this work.

Reply:
Coming back to this after a bit of time to think it through: I'm still a little torn so I'll leave it to you.

The problematic point for me is that this change will silence an issue in CI if `-CheckSymbols` won't be passed (it's conditionally done so right now). Also, it wipes the differentiation we had between configuring the project (in `update-sentry.ps` and building it).

Reply:
I see your point.
I've updated the PR to, instead of passing the -CheckSymbol to the `Builder` it instead gets passed to the `sentry_update`. So we only set up the cli options to try to upload symbols if we're actually checking for them.